http/headers/content-encoding.json

```
@@ -78,7 +78,7 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "11"
```

Proposing some note text based on https://github.com/mdn/browser-compat-data/pull/10330#issuecomment-839978678

```suggestion
                "version_added": "11",
                "notes": "Unsupported before macOS 10.13 High Sierra."
```